releng/travis-script.sh

```diff
@@ -76,6 +76,40 @@ esac
 set -o xtrace
 make print-vars
 
+case "$TRAVIS_EVENT_TYPE" in
```

Comment:
Any reason this isn't just `if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]` ?

Also, should the cronjob continue to run the build when it's a cronjob, or exit after the Envoy checks?

Reply:
> Any reason this isn't just if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]] ?

Well, just in case we want to add other events for `TRAVIS_EVENT_TYPE` later on, one of `push, pull_request, api, cron`.

> Also, should the cronjob continue to run the build when it's a cronjob, or exit after the Envoy checks?

> Also, should the cronjob continue to run the build when it's a cronjob, or exit after the Envoy checks?

Makes sense, I've made it exit after envoy checks.

Reply:
Where? I'm not seeing that. 😕

Reply:
Sorry about that :\ I had made the changes to the main `make test` instead of here. But anyway, fixed it now. It should now exit after running Envoy tests.